app/views/shared/_zero_results.html.erb

```diff
@@ -1,44 +1,49 @@
-
+  <div class="panel-body zero-results">
+    <h3><%= t 'blacklight.search.zero_results.modify_your_search', search_type: search_type_name %></h3>
       <% if from_advanced_search? %>
-
         <li>
           <i class="fa fa-search-plus"></i>
           <%= link_to t('blacklight.search.zero_results.return_to_advanced_search'), "JavaScript:history.back();" %>
         </li>
         <li><%= t 'blacklight.search.zero_results.limit' %></li>
 
       <% else %>
-
-        <li>
-         <%= t 'blacklight.search.zero_results.your_search' %>
-         <% if @search_modifier.has_query? %>
-           <%= "#{search_field_label(params)}: #{params[:q]}" %>
-         <% end %>
-         <%= @search_modifier.selected_filter_labels %>
-       </li>
-       <% if @search_modifier.has_filters? && @search_modifier.has_query? %>
-         <li><%= t 'blacklight.search.zero_results.limit_fields' %>
-           <%= link_to "#{search_field_label(params)}: #{params[:q]}", search_catalog_path(@search_modifier.params_without_filters), data: {behavior: "backend-lookup", lookup: "#{catalog_backend_lookup_path(@search_modifier.params_without_filters)}"} %>
-         </li>
-       <% end %>
-       <% if @search_modifier.fielded_search? %>
-         <li><%= t 'blacklight.search.zero_results.search_fields' %>
-           <%= link_to "All fields: #{params[:q]}", search_catalog_path(@search_modifier.params_without_fielded_search_and_filters), data: {behavior: "backend-lookup", lookup: "#{catalog_backend_lookup_path(@search_modifier.params_without_fielded_search_and_filters)}"} %>
-         </li>
-       <% end %>
-
-       <% end %>
-
-   </ul>
+        <dl>
+          <dt>
+            <%= t 'blacklight.search.zero_results.your_search' %>
+          </dt>
+          <dd>
+            <% if @search_modifier.has_query? %>
+              <%= "#{search_field_label(params)} > #{params[:q]}" %>
+            <% end %>
+            &nbsp;+ <%= @search_modifier.selected_filter_labels %>
+          </dd>
+          <% if @search_modifier.has_filters? && @search_modifier.has_query? %>
+            <dt>
+              <%= t 'blacklight.search.zero_results.limit_fields' %>
+            </dt>
+            <dd>
+              <%= link_to "#{search_field_label(params)} > #{params[:q]}", search_catalog_path(@search_modifier.params_without_filters), data: {behavior: "backend-lookup", lookup: "#{catalog_backend_lookup_path(@search_modifier.params_without_filters)}"} %>
+            </dd>
+          <% end %>
+          <% if @search_modifier.fielded_search? %>
+            <dt>
+              <%= t 'blacklight.search.zero_results.search_fields' %>
+            </dt>
+            <dd>
+              <%= link_to "#{params[:q]}", search_catalog_path(@search_modifier.params_without_fielded_search_and_filters), data: {behavior: "backend-lookup", lookup: "#{catalog_backend_lookup_path(@search_modifier.params_without_fielded_search_and_filters)}"} %>
```

Comment:
Unnecessary quote on the first argument and on the `lookup` option. Please add spaces around hash-literal braces for consistency with other uses. See https://github.com/bbatsov/ruby-style-guide#spaces-braces